Stop raising exception when purely absorbing `MolecularAtmosphere` is mixed with `ParticleLayer` (#239)
eradiate/eradiate: CHANGELOG.md

* Some dependencies are now optional, although recommended ({ghpr}`266`).
* Added new sahara and continental particle radiative properties including the
  full scattering phase matrix ({ghpr}`259`).

> * Stop raising an exception when a `HeterogeneousAtmosphere` is used to mix a
>   purely absorbing `MolecularAtmosphere` with a `ParticleLayer` ({ghpr}`239`).

Reviewer: Let's explain that with a higher-level view:
```suggestion
* Allow mixing a purely absorbing `MolecularAtmosphere` with a `ParticleLayer` 
  ({ghpr}`239`).
```
